docs/user_guide/serving.md

```diff
@@ -20,31 +23,29 @@ The structure of the input to the REST API and the structure of the output that
 For each input of the model, the predict endpoint expects a field with a name.
 For instance, a model trained with an input text field named `english_text` would expect a POST like:
 
-```
+```bash
 curl http://0.0.0.0:8000/predict -X POST -F 'english_text=words to be translated'
 ```
 
 If the model was trained with an input image field, it will instead expects a POST with a file, like:
 
-```
+```bash
 curl http://0.0.0.0:8000/predict -X POST -F 'image=@path_to_image/example.png'
 ```
 
 A model with both a text and an image field will expect a POST like:
 
-```
+```bash
 curl http://0.0.0.0:8000/predict -X POST -F 'text=mixed together with' -F 'image=@path_to_image/example.png'
 ```
 
 ### Output format
 
-The output format is  a JSON that is independent of the number of inputs and their data types, it only depends on the number of outputs the model pipeline was trained to predict and their data types.
-
-At the moment, Ludwig can predict binary, numerical, categorical, set, sequence and text outputs.
+The output format is a JSON that depends on the number and types of outputs the model pipeline was trained to predict.
 
 For binary outputs, the JSON structure returned by the REST PI is the following:
```

Comment:
s/REST PI/REST API    (or REST Endpoint)  .  Find-replace in the file, this typo is copy/pasted repeatedly.

Reply:
Done.